AWSAuthSDK/Sources/AWSAuthCore/AWSUIConfiguration.h

```diff
@@ -17,16 +17,28 @@
 
 /**
  @property logoImage
- @brief The logo to be used on the Auth UI Screen
+ @brief The logo to be used on the Auth UI Screen.
  **/
 @property (nonatomic, nullable) UIImage *logoImage;
 
 /**
  @property backgroundColor
- @brief Gets the backgorund color of the sign in screen configured by the user
+ @brief The background color of the sign-in screen.
```

Comment:
I see you have some use cases that are only available on iOS 13. Please document how it is used.

Reply:
Good call, it will be documented on the next revision.

I'll also update our docsite after this PR is merged.